src/styles/core/colors/_light.scss

```diff
@@ -190,4 +190,9 @@
   --color-tutorial-hero-text: #{dark-color(figure-gray)};
   --color-tutorial-hero-background: #{dark-color(fill)};
   --color-navigator-item-hover: #{change-color(light-color(fill-blue), $alpha: 0.05)};
+
+  --color-card-background: var(--color-fill);
```

Comment:
This is a little hard to say without some example data, but is it an expected scenario to have a card without an image? Depending on how common that scenario is, we might want to consider eliminating special color values for them to avoid bloating the very large number of special colors we already have. Ignoring the shadow, the only benefit to including these new variables would be so that these could potentially be specialized on their own through theming, right?

Not necessarily advocating for one approach of the other yet—just making sure we've thought through the common use case before we add new global color variables.